### DISCUSSION TITLE
Will v0.17.3 be released to align with buffalo v0.17.3

### DISCUSSION THREAD

**Discussion by jessehu** [gobuffalo/cli#31] [General]

**Comment by paganotoni** [comment]
Hey @jessehu, Thanks for asking. 

There is an upcoming release of the Buffalo CLI but its not intended to align with the Buffalo library, since we moved the CLI to a separate repo the CLI and the Buffalo Library have different stories.

**Reply by jessehu** [reply]
Thanks @paganotoni. Current CLI v0.17.2 generates the Dockerfile with gobuffalo/buffalo:v0.17.2 which doesn't work Go 1.17 (but 1.16 works). I'm wondering new CLI v0.17.3 will use gobuffalo/buffalo:v0.17.3. Anyway it's not a blocker issue. There is workaround.

Buffalo is really a really cool tool, especially for the engineers who are familiar with Rails.
